.github/workflows/ci-pipeline.yaml

```
@@ -17,6 +17,7 @@ jobs:
   get-chart:
     runs-on: ubuntu-latest
     name: 'Get modified charts'
+    if: github.event.pull_request.state != 'closed'
```

In order to maintain the style in the file, we usually use `${{ <expression> }}` for conditionals that can be expressed as one-liners:

```suggestion
    if: ${{ github.event.pull_request.state != 'closed' }}
```